pgscatalog.match/src/pgscatalog/match/lib/_match/plink.py

```diff
@@ -17,18 +17,20 @@ def plinkify(df):
 
     Some fake match data where effect_allele is the same but other_allele is different:
 
-    >>> data = {'row_nr': [1, 2, 3], 'chr_name': ['11', '11', '11'], 'chr_position': [69331418, 69331418, 69331419], 'effect_allele': ['A', 'C', 'T'], 'other_allele': ['C', 'C', 'T'], 'effect_weight': ['1', '1.5', '2'], 'effect_type': ['additive', 'additive', 'additive'], 'accession': ['pgs1', 'pgs2', 'pgs3'],'ID': ['11:69331418:A:C', '11:69331418:A:C', '11:69331419:T:T'],'matched_effect_allele': ['A', 'C', 'T']}
+    >>> data = {'row_nr': [1, 1, 1], 'chr_name': ['11', '11', '11'], 'chr_position': [69331418, 69331418, 69331419], 'effect_allele': ['A', 'C', 'T'], 'other_allele': ['C', 'C', 'T'], 'effect_weight': ['1', '1.5', '2'], 'effect_type': ['additive', 'additive', 'additive'], 'accession': ['pgs1', 'pgs2', 'pgs3'],'ID': ['11:69331418:A:C', '11:69331418:A:C', '11:69331419:T:T'],'matched_effect_allele': ['A', 'C', 'T']}
     >>> plinked = plinkify(pl.DataFrame(data).lazy())
     >>> dfs = sorted((x.collect() for x in plinked[EffectType.ADDITIVE]), key= lambda x: len(x))
-    >>> assert len(dfs) == 2
+    >>> len(dfs)
+    2
+
     >>> dfs[0].select(["row_nr", "accession", "ID", "matched_effect_allele"])
     shape: (1, 4)
     ┌────────┬───────────┬─────────────────┬───────────────────────┐
     │ row_nr ┆ accession ┆ ID              ┆ matched_effect_allele │
     │ ---    ┆ ---       ┆ ---             ┆ ---                   │
     │ i64    ┆ str       ┆ str             ┆ str                   │
     ╞════════╪═══════════╪═════════════════╪═══════════════════════╡
-    │ 2      ┆ pgs2      ┆ 11:69331418:A:C ┆ C                     │
+    │ 1      ┆ pgs2      ┆ 11:69331418:A:C ┆ C                     │
     └────────┴───────────┴─────────────────┴───────────────────────┘
 
     >>> dfs[1].select(["row_nr", "accession", "ID", "matched_effect_allele"])
```

Comment:
It might make more sense for the [0] data frame to have more variants than [1], that's how it was before and the longest jobs will then be submitted first.

Reply:
good point!